Source/WebKit/WebProcess/GPU/GPUProcessConnection.cpp

```diff
@@ -188,10 +188,15 @@ void GPUProcessConnection::didReceiveInvalidMessage(IPC::Connection&, IPC::Messa
 SampleBufferDisplayLayerManager& GPUProcessConnection::sampleBufferDisplayLayerManager()
```

Comment:
I added a protected version of this to fix the one and only call site that became unsafe as a result of `SampleBufferDisplayLayerManager` now being ref-counted. That means that this non-protected `sampleBufferDisplayLayerManager()` is called only in the protected version and no where else. Should we still keep this non-protected version the way I have here?

Reply:
Yes, I would keep in case someone needs it in the future.